packages/sdk/lib/node.ts

```diff
@@ -27,35 +27,39 @@ export {
 };
 
 // Itly Node SDK is the same as base (for now)
-export class ItlyNode {
+export class Itly {
```

Comment:
Since base/Itly and node/Itly are identical can you make this re-export base Itly unchanged with the other classes rather than making a subclass. Keep the default export `export default Itly`

Reply:
It seems, node.ts is unnecessary - I've removed it

Reply:
Even though it is identical to base we still want to keep node.ts to allow for [better import naming in TypeScript for strong typing](https://github.com/iterativelyhq/itly-sdk#sdk-types-for-typescript). Can you please re-add it.

Reply:
Done